src/contacts/processors/impl/SearchBackRecordProcessor.java

```diff
@@ -0,0 +1,17 @@
+package contacts.processors.impl;
+
+import contacts.processors.IActionProcessor;
+
+import java.io.IOException;
+
+public class SearchBackRecordProcessor implements IActionProcessor {
```

Comment:
I think all such processors must be deleted

Reply:
Done, deleted